<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Template</class>
 <widget class="QWidget" name="Template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Template</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="u_titleLabel">
       <property name="text">
        <string>Select a template</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,3">
     <item>
      <widget class="QTreeView" name="u_templateTree"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="2,5">
       <item>
        <widget class="QTextBrowser" name="u_detailsTB"/>
       </item>
       <item>
        <widget class="QWidget" name="u_configWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="u_editLabel">
              <property name="text">
               <string>Enter details</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeView" name="u_configTreeView"/>
          </item>
          <item>
           <widget class="QCheckBox" name="u_lowercaseCheckBox">
            <property name="text">
             <string>Generated filenames in  all lowercase</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="u_locationLineEdit">
              <property name="placeholderText">
               <string>Copy files to folder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="u_locationToolButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="u_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>u_templateTree</tabstop>
  <tabstop>u_detailsTB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
